### ISSUE TITLE
how to suppress some training output?

### ISSUE THREAD

**Issue by archie9211** [AlexeyAB/darknet#6186]
there are bunch of lines while training spamming on the console. 
I want to suppress those and only show following lines:

```

 Tensor Cores are disabled until the first 3000 iterations are reached.
 (next mAP calculation at 1000 iterations)
 43: 1294.019287, 1107.934326 avg loss, 0.000000 rate, 7.805518 seconds, 2752 images, 2482.883835 hours left
Loaded: 0.000025 seconds
Tensor Cores are disabled until the first 3000 iterations are reached.
 (next mAP calculation at 1000 iterations)
 43: 1294.019287, 1107.934326 avg loss, 0.000000 rate, 7.805518 seconds, 2752 images, 2482.883835 hours left
Loaded: 0.000025 seconds
Tensor Cores are disabled until the first 3000 iterations are reached.
 (next mAP calculation at 1000 iterations)
 43: 1294.019287, 1107.934326 avg loss, 0.000000 rate, 7.805518 seconds, 2752 images, 2482.883835 hours left
Loaded: 0.000025 seconds

```
I want to remove such code from output as below

```

v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 139 Avg (IOU: 0.000000, GIOU: 0.000000), Class: 0.000000, Obj: 0.000000, No Obj: 0.000000, .5R: 0.000000, .75R: 0.000000, count: 1, class_loss = 0.000000, iou_loss = 0.000000, total_loss = 0.000000
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 150 Avg (IOU: 0.731456, GIOU: 0.728674), Class: 0.999806, Obj: 0.064629, No Obj: 0.000482, .5R: 1.000000, .75R: 0.333333, count: 3, class_loss = 1.723221, iou_loss = 0.691870, total_loss = 2.415092
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 161 Avg (IOU: 0.764929, GIOU: 0.760924), Class: 0.999823, Obj: 0.278181, No Obj: 0.004199, .5R: 1.000000, .75R: 0.666667, count: 6, class_loss = 2.220311, iou_loss = 0.476424, total_loss = 2.696735
 total_bbox = 15853129, rewritten_bbox = 0.122127 %
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 139 Avg (IOU: 0.000000, GIOU: 0.000000), Class: 0.000000, Obj: 0.000000, No Obj: 0.000000, .5R: 0.000000, .75R: 0.000000, count: 1, class_loss = 0.000000, iou_loss = 0.000000, total_loss = 0.000000
v3 (iou loss, Normalizer: (iou: 0.07, cls:
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 139 Avg (IOU: 0.362766, GIOU: 0.302798), Class: 0.408631, Obj: 0.518220, No Obj: 0.475449, .5R: 0.264706, .7v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 139 Avg (IOU: 0.423291, GIOU: 0.359377), Class: 0.419097, Obj: 0.491668, No Obj: 0.475477, .5R: 0.375000, .75R: 0.012500, count: 80, class_loss = 4126.025391, iou_loss = 248.927734, total_loss = 4374.953125
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 150 Avg (IOU: 0.383353, GIOU: 0.352415), Class: 0.496117, Obj: 0.516649, No Obj: 0.506771, .5R: 0.217822, .75R: 0.029703, count: 101, class_loss = 1197.082031, iou_loss = 54.005249, total_loss = 1251.087280

```

**Comment by stephanecharette** [comment]
You have two options:

- edit the darknet source and search for every `printf()` call, keeping only the ones you want
- when you call darknet, send both STDOUT and STDERR to a file or pipe, and grep for only the lines you want

Example of that 2nd solution:

    darknet detector ...etc... 2>&1 | grep "iou loss, Normalizer"

**Comment by minhtcai** [comment]
I think you can put 2>/dev/null at the end of the command.
./darknet detector train .... 2>/dev/null

**Comment by archie9211** [comment]
> You have two options:
> 
> * edit the darknet source and search for every `printf()` call, keeping only the ones you want
> * when you call darknet, send both STDOUT and STDERR to a file or pipe, and grep for only the lines you want
> 
> Example of that 2nd solution:
> 
> ```
> darknet detector ...etc... 2>&1 | grep "iou loss, Normalizer"
> ```

Thank you. Second option worked for me...
i used sed : 
`darknet detector train .....  2>&1  | sed -e "/iou loss/d" -e "/total_bbox/d"`

**Comment by xiaobumiDM** [comment]
I want to ask you that “Tensor Cores are disabled until the first 3000 iterations are reached.”  if this question effect your training result? Thank you.

**Comment by archie9211** [comment]
Training result?? You mean performance of the model?? No,

Training speeds up after tensor cores are used by very small fraction.
